data-types.md

```diff
@@ -0,0 +1,291 @@
+
+
+```
+"columns": [
+    {
+        "title": "",
+        "name": "",
+        "datatype": "float"
+    }
+]
+```
+
+**Note** Never use float. As you cannot guarantee the accuracy.
+
+**Note** Beaware that different machines offer different calculations.
+
+## Boolean
+
+Boolean has the value space required to support the mathematical concept of binary-valued logic.
+
+Examples include - True and False.
+
+```
+"columns": [
+    {
+        "title": "",
+        "name": "",
+        "datatype": "boolean"
+    }
+]
+```
+
+**NOTE** : You need to be aware of whitespace. In both your csv and JSON file.
+
+# Tables
+
+When writing your JSON-LD file to accompany your csv, you can write it in a way to include multiple csvs.
+
+Below is an extract of csv that will be described in the JSON-LD.
+
+| period_type | period_code | period_label | area_code | area_label | area_type | observation | measure | unit | lcl_95 | ucl_95 |     obs_status |
+| ----------- | ----------- | ------------ | --------- | ---------- | --------- | ----------- | ------- | ---- | ------ | ------ | ---------- | 
+| government-year | 2011-2012 | 2011-2012 | K02000001 | United Kingdom | Country | 3.13 | Anxiety mean score (where 10 is 'completely anxious') | Unitless | 3.11 | 3.15 |
+| government-year | 2022-2023 | 2022-2023 | K02000001 | United Kingdom | Country | 3.23 | Anxiety mean score (where 10 is 'completely anxious') | Unitless | 3.2 | 3.26 |
+| government-year | 2021-2022 | 2021-2022 | K02000001 | United Kingdom | Country | 3.12 | Anxiety mean score (where 10 is 'completely anxious') | Unitless | 3.1 | 3.15 |
+
+
+In order to write a JSON-LD to include mulitple csvs. You need to include the csvs column details within a tables [ ] section. As described below.
+
+You will need to use { } for each csv that you want to describe. Within these sections you should include a url which will contain the name of your csv. A table schema section  is next, where you describe the columns of your csv.
+
+Each column of the csv will need to be described. Each entry should include a title, name and data type. The title provies the human readable name of the column. The name provides the machine readable name of the column. This should be in `snake_case`. The data type provides the detail of what data type is being represented in the column.
+
+There are additional descriptors that you can add to your column sections. These include:
+
+- `suppressOutput` this will enable you to suppress the column from view.
+- `valueurl` will contain a url.
+- `propertyuri` will contain the `schema:url`.
+- `label` this can be used for additional context.
+
+
+```
+{
+                    "title": "",
+                    "name": "",
+                    "data_type": "",
+                    "suppressOutput": true
+                },
+```
+
+The right way to describe each column is shown below:
+
+```
+"tables": [
+    {
+        "url": "anxiety.csv",
+        "tableSchema": {
+            "columns": [
```

Comment:
Breaking down the example "how to encode time periods to be both human readable and machine readable", walk people through the process of combining suppressingOutput and virtual columns.

Reply:
Done.